lab/src/Microsoft.ServiceModel.Syndication/tests/Microsoft.ServiceModel.Syndication.Tests.csproj

```diff
@@ -27,8 +27,35 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="TestFeeds\atom_spec_example.xml">
```

Comment:
They seem to work, will edit.

---
In reply to: [126848284](https://github.com/dotnet/wcf/pull/2028#discussion_r126848284) [](ancestors = 126848284)